<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="GB_Data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>데이터 처리</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>훈련 데이터 셋        :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>118</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>검증 데이터 셋        :</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_TRAINING_DATA_SET_SIZE">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_VALIDATION_DATA_SET_SIZE">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>118</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_LOAD_TRAINING_SET">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>불러오기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_LOAD_VALIDATION_SET">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>불러오기</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_MODEL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>761</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>모델</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="TE_MODELCHECK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>631</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_RESNET18">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ResNet-18</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_RESNET34">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ResNet-34</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_RESNET50">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>110</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ResNet-50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_RESNET101">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>150</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ResNet-101</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_RESNET152">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>190</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ResNet-152</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_VGG16">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>230</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>VGG-16</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_VGG19">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>270</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>VGG-19</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_MOBILENET_V1">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>310</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MobileNet_V1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_MOBILENET_V2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>350</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MobileNet_V2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_PROGRESS">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>진행상황</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QGroupBox" name="GB_PROGRESS_TRAINING">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>학습</string>
     </property>
     <widget class="QLabel" name="LB_TRAINING_EPOCH_CURRENT">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_EPOCH_TOTAL">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Loss                    : </string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_EPOCH_LOSS">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Accuracy (prec1)     : </string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_EPOCH_ACCURACY">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_IDX_CURRENT">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_IDX_TOTAL">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_EPOCH_CURRENT_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Epoch :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_TRAINING_EPOCH_CURRENT_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Training    :</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GB_PROGRESS_VALIDATION">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>검증</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Loss                    : </string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_VALIDATION_EPOCH_LOSS">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Accuracy (prec1)     : </string>
      </property>
     </widget>
     <widget class="QLabel" name="LB_VALIDATION_EPOCH_ACCURACY">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="LB_TRAINING_EPOCH_ACCURACY_PREC5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Accuracy (prec5)    : </string>
     </property>
    </widget>
    <widget class="QLabel" name="LB_VALIDATION_EPOCH_ACCURACY_PREC5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Accuracy (prec5)     : </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_OPTIMIZER">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>440</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Optimizer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="BTN_OPTIMIZER_SGD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>SGD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_OPTIMIZER_ADAM">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Adam</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_OPTIMZER_ADAGRAD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Adagrad</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_OPTIMZER_RMSPROP">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>RMSProp</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GB_TRAINING_PARAMETER_SETTING">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>학습 파라메타 설정</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Learning Rate                  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Epoch                              :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Batch Size                       :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_TRAINING_PARAMETER_SETTING_SAVE">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PARAMETER_LEARNING_RATE">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1e-5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PARAMETER_EPOCH">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PARAMETER_BATCH_SIZE">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>64</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BTN_TRAIN">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>학습하기</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
